indexing-service/src/main/java/io/druid/indexing/overlord/ImmutableWorkerInfo.java

```diff
@@ -74,6 +74,11 @@ public int getCurrCapacityUsed()
     return availabilityGroups;
   }
 
+  @JsonProperty("availableCapacity")
```

Comment:
Shouldn't have `@JsonProperty` annotation, as this is a derivative of other fields and the class doesn't expect this property in constructor